script/data.js

```diff
@@ -0,0 +1,143 @@
+    cardTitle: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    company: 'CANOPY',
+    cardTitle2: 'TONIC',
+    description2:
+      'A daily selection of privately personalized; no accounts or sign-ups required.',
+    company2: 'CONOPY',
+    no: 'detail1',
+    cardImage1: './images/Mobpics2.png',
+    counterText: 'Back End Dev',
+    year: '2015',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+  },
+
+  {
+    id: '3',
+    cardTitle: 'TONIC',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    company: 'CANOPY',
+    cardTitle2: 'TONIC',
+    description2:
+      'A daily selection of privately personalized; no accounts or sign-ups required.',
+    company2: 'CONOPY',
+    no: 'detail1',
+    cardImage1: './images/Mobpics3.png',
+    counterText: 'Back End Dev',
+    year: '2015',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+  },
+
+  {
+    id: '4',
+    cardTitle: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    company: 'CANOPY',
+    cardTitle2: 'TONIC',
+    description2:
+      'A daily selection of privately personalized; no accounts or sign-ups required.',
+    company2: 'CONOPY',
+    no: 'detail1',
+    cardImage1: './images/Mobpics4.png',
+    counterText: 'Back End Dev',
+    year: '2015',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+  },
+];
+
+// Desktop Version//
+export const desktopProjects = [
+  {
+    // card 1 details
+    id: '1',
+    cardTitle: 'TONIC',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    company: 'CANOPY',
+    cardTitle2: 'TONIC',
+    description2:
+      'A daily selection of privately personalized; no accounts or sign-ups required.',
+    company2: 'CONOPY',
+    no: 'detail1',
+    cardImage1: './images/desktop1.png',
+    counterText: 'Back End Dev',
+    year: '2015',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+  },
+
+  {
+    id: '2',
+    cardTitle: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    company: 'FACEBOOK',
+    cardTitle2: 'TONIC',
+    description2:
+      'A daily selection of privately personalized; no accounts or sign-ups required.',
+    company2: 'CONOPY',
+    no: 'detail1',
+    cardImage1: './images/desktop2.png',
+    counterText: 'Full Stack Dev',
+    year: '2015',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+  },
+
+  {
+    id: '3',
+    cardTitle: 'TONIC',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    company: 'FACEBOOK',
+    cardTitle2: 'Facebook 360',
+    description2:
+      'A daily selection of privately personalized; no accounts or sign-ups required.',
```

Comment:
- [ ] Why are you duplication the project data here?  Dont you think the same project for mobile is for desktop so make sure you keep your code DRY by creating only one array of your project details :blush:

Reply:
@Desmond Owusu
 Hi, the mobile and the desktop popups use two different data structures for template one. Can you please give me an idea on how I can implement this for the two screens using only one data structure?